tests/components/matrix/test_send_message.py

```diff
@@ -23,24 +27,53 @@ async def test_send_message(
     await matrix_bot._login()
 
     # Send a message without an attached image.
+    matrix_bot._handle_multi_room_send.reset_mock()
     data = {ATTR_MESSAGE: "Test message", ATTR_TARGET: list(TEST_JOINABLE_ROOMS)}
     await hass.services.async_call(
         MATRIX_DOMAIN, SERVICE_SEND_MESSAGE, data, blocking=True
     )
 
+    expected_data = {
+        "target_rooms": list(TEST_JOINABLE_ROOMS.keys()),
+        "message_type": "m.room.message",
+        "content": {"msgtype": "m.text", "body": data[ATTR_MESSAGE]},
+    }
+    matrix_bot._handle_multi_room_send.assert_called_once_with(
+        target_rooms=expected_data["target_rooms"],
+        message_type=expected_data["message_type"],
+        content=expected_data["content"],
+    )
```

Comment:
Use dict unpacking syntax
```suggestion
    matrix_bot._handle_multi_room_send.assert_called_once_with(**expected_data)
```